README.md

```diff
@@ -1,4 +1,5 @@
 # pythonparser
+[![elena-lyulina](https://circleci.com/gh/elena-lyulina/pythonparser/tree/inverse_parser.svg?style=shield)](https://app.circleci.com/pipelines/github/elena-lyulina/pythonparser?branch=inverse_parser)
 
 This repo contains parsers from **python code** to **xml/json** and vice versa.
 There are parsers for **python2** (see [pythonparser](src/main/python/pythonparser-2.py), source code from [this repo](https://github.com/GumTreeDiff/pythonparser)) and **python3** (see [pythonparser3](src/main/python/pythonparser-3.py), source code from [this repo](https://github.com/Varal7/pythonparser) and [this project](https://eth-sri.github.io/py150)). 
```

Comment:
It's my fault cause I've wrote the beginning of readme as an example :)
Totally agree with replacing repo -> repository, but not sure about marking only one word. It's no big deal for me, but it would be nice to see some sources you rely on.

Reply:
I think you are right and I prefer this style, but it is not necessary for general I think, thank you!